ush/config_defaults.sh

```diff
@@ -425,6 +425,45 @@ FCST_LEN_HRS_CYCLES=( )
 DA_CYCLE_INTERV="3"
 RESTART_INTERVAL="3,6"
 #
+#-------------------------------------------------------------------------------------
+#      GSI Namelist parameters configurable across differnt applications
+# if we need to tune one GSI namelist parameter, we can elevate it to a shell variable
+# and assign value in config.sh and give it a default value in config_default.sh
+# In realtime testing, don't need to regenerate the whole workflow, you can tweak 
+# $EXPTDIR/var_defns.sh and $USH/template/gsiparm.anl.sh to make sure the change is
+# expected and then put it back into config.sh and config_default.sh
+#       (need to follow FORTRAN namelist convetion)
+#-------------------------------------------------------------------------------------
+# &SETUP  and &BKGERR
+diag_radardbz=.true.
+write_diag_2=.false.
+bkgerr_vs=1.0
+bkgerr_hzscl=0.373,0.746,1.5   #no trailing ,
+
+# &HYBRID_ENSEMBLE
+readin_localization=.true.     #if true, it overwrites the "beta1_inv/ens_h/ens_v" setting
+beta1_inv=0.15                 #beata_inv is 1-ensemble_wgt
+ens_h=110
+ens_v=3
+regional_ensemble_option=1     #1 for GDAS
+grid_ratio_ens=3               #analysis 3km, so ensemble=3*3=9km. GDAS ensemble is 20km
+i_en_perts_io=1                #0 or 1: original file   3: pre-processed ensembles
+
+# &RAPIDREFRESH_CLDSURF
+l_PBL_pseudo_SurfobsT=.false.
+l_PBL_pseudo_SurfobsQ=.false.
+i_use_2mQ4B=0
+i_use_2mT4B=0
+#-----------------------------------------------------------------------
+#
+ANAVINFO_FN="anavinfo.fv3lam_hrrr"
+CONVINFO_FN="convinfo.rrfs"
+BERROR_FN="rap_berror_stats_global_RAP_tune" #under $FIX_GSI
+OBERROR_FN="errtable.rrfs"
+HYBENSINFO_FN="hybens_info.rrfs"
+AIRCRAFT_REJECT="/home/amb-verif/acars_RR/amdar_reject_lists"
+SFCOBS_USELIST="/home/amb-verif/ruc_madis_surface/mesonet_uselists"
+#
```

Comment:
The standard convention we use here is that global configuration settings are capitalized. Please follow that standard for these new options.

Reply:
Do you refer to those GSI namelist variables?  Currently I intentionally keep it the same as the original namelist parameter names but they can be changed easily if we decide that changing them to capitalized is better.

Reply:
Since we will hopefully approach a chance to the management of the namelist in the near future, I'd say leave these for now. I can certainly help with that set of modifications.